test/testutils.js

```diff
@@ -420,24 +420,43 @@ testUtils.assertStylesheet = function assertStylesheet(
  * Injecting content into a fixture and return queried element within fixture
  *
  * @param {String|Node} content to go into the fixture (html or DOM node)
+ * @param {String?} query - the CSS selector query to find target DOM node
  * @return HTMLElement
  */
 testUtils.queryFixture = function queryFixture(html, query) {
+  query = query || '#target';
   var rootNode = testUtils.fixtureSetup(html);
+  var vNode = axe.utils.querySelectorAll(rootNode, query)[0];
+  assert.exists(
+    vNode,
+    'Node does not exist in query `' +
+      query +
+      '`. This is usually fixed by adding the default target (`id="target"`) to your html parameter. If you do not intend on querying the fixture for #target, consider using `axe.testUtils.fixtureSetup()` instead.'
+  );
   return axe.utils.querySelectorAll(rootNode, query || '#target')[0];
```

Comment:
```suggestion
  return vNode;
```

Reply:
Done, and updated JSDoc to `@return {VirtualNode}`